SConstruct

```diff
@@ -112,6 +112,7 @@ opts.Add(EnumVariable('target', "Compilation target", 'debug', ('debug', 'releas
 opts.Add(EnumVariable('optimize', "Optimization type", 'speed', ('speed', 'size')))
 opts.Add(BoolVariable('tools', "Build the tools (a.k.a. the Godot editor)", True))
 opts.Add(BoolVariable('use_lto', 'Use link-time optimization', False))
+opts.Add(BoolVariable('use_thinlto', "Use ThinLTO (implies 'use_lto=yes')", False))
```

Comment:
It shouldn't be added here if it's specific to one platform. Adding it in `platform/x11/detect.py` is sufficient.

Reply:
Ok I removed it